doc/src/modules/physics/continuum_mechanics/beam_problems.rst

```diff
@@ -203,3 +203,170 @@ is applied from the mid til the end of the beam.
                                                           8         120                               
     ──────────────────────────────────────────────────────────────────────────────────────────────────
                                                    E⋅I                                                
+
+Example 4
+---------
+
+An overhanging beam of length 8 meters is pinned at 1 meter from starting point
+and supported by a roller 1 meter before the other end. It is subjected
+to a distributed constant load of 10 KN/m from the starting point till
+2 meters away from it. Two pointloads of 20KN and 8KN are applied at
+5 meters and 7.5 meters away from the starting point respectively.
```

Comment:
Did you come up with this wording or is it copied from a book, etc?

Reply:
These were specific questions like "Find max deflection in the beam " along with a diagram given. 
So yeah, I wrote the wording on my own.

Reply:
Ok, that's good then, we just want to abide by copyright rules.